newton/_src/utils/import_usd.py

```
@@ -1231,9 +1240,15 @@ def warn_invalid_desc(path, descriptor) -> bool:
                     if verbose:
                         print(f"Sorting joints using {joint_ordering} ordering...")
-                    sorted_joints = topological_sort(
+                    sorted_joints, reversed_joint_list = topological_sort_undirected(
                         joint_edges, use_dfs=joint_ordering == "dfs", ensure_single_root=True
                     )
+                    if reversed_joint_list:
+                        reversed_joint_paths = [joint_names[joint_id] for joint_id in reversed_joint_list]
+                        reversed_joint_names = ", ".join(reversed_joint_paths)
+                        raise ValueError(
+                            f"Reversed joints are not supported: {reversed_joint_names}. Ensure that the joint parent body is defined as physics:body0 and the child is defined as physics:body1 in the joint prim."
+                        )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)